src/api/Shrooms.Presentation.Api/Web.config

```diff
@@ -1,342 +1,343 @@
-    </targets>
-    <rules>
-      <logger name="Hangfire.*" maxlevel="Warn" writeTo="blackHole" final="true" />
-      <logger name="*" minlevel="Trace" writeTo="f1" />
-      <logger name="*" minlevel="Trace" writeTo="aiTarget" />
-    </rules>
-    <extensions>
-      <add assembly="Microsoft.ApplicationInsights.NLogTarget" />
-    </extensions>
-  </nlog>
-  <entityFramework>
-    <defaultConnectionFactory type="System.Data.Entity.Infrastructure.SqlConnectionFactory, EntityFramework" />
-    <providers>
-      <provider invariantName="System.Data.SqlClient" type="System.Data.Entity.SqlServer.SqlProviderServices, EntityFramework.SqlServer" />
-    </providers>
-  </entityFramework>
-  <features>
-    <add name="Monitors" state="Disabled" />
-    <add name="Impersonation" state="Disabled" />
-  </features>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <configSections>
+    <section name="system.identityModel" type="System.IdentityModel.Configuration.SystemIdentityModelSection, System.IdentityModel, Version=4.0.0.0, Culture=neutral, PublicKeyToken=B77A5C561934E089" />
+    <section name="system.identityModel.services" type="System.IdentityModel.Services.Configuration.SystemIdentityModelServicesSection, System.IdentityModel.Services, Version=4.0.0.0, Culture=neutral, PublicKeyToken=B77A5C561934E089" />
+    <section name="entityFramework" type="System.Data.Entity.Internal.ConfigFile.EntityFrameworkSection, EntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" requirePermission="false" />
+    <section name="resizer" type="ImageResizer.ResizerSection" requirePermission="false" />
+    <section name="features" type="ReallySimpleFeatureToggle.Configuration.AppConfigProvider.FeatureConfigurationSection, ReallySimpleFeatureToggle" />
+    <section name="RegisteredOrganizations" type="System.Configuration.NameValueFileSectionHandler, System, Version=1.0.3300.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
+    <section name="nlog" type="NLog.Config.ConfigSectionHandler, NLog" />
+  </configSections>
+  <!--
+    Mailing Service Settings,
+    you can use services like SendGrind, for more info visit https://sendgrid.com/
+    for testing purposes you can use https://mailtrap.io/
+  -->
+  <system.net>
+    <mailSettings>
+      <smtp from="noreply@simoona.com">
+        <network host="smtp.mailtrap.io" userName="" password="" port="2525" />
+      </smtp>
+    </mailSettings>
+  </system.net>
+  <!-- App Settings -->
+  <appSettings>
+    <add key="IsProductionBuild" value="false" />
+    <add key="webpages:Version" value="3.0.0.0" />
+    <add key="webpages:Enabled" value="false" />
+    <add key="PreserveLoginUrl" value="true" />
+    <add key="UnobtrusiveJavaScriptEnabled" value="true" />
+    <add key="CorsOrigin" value="*" />
+    <add key="EmailEnabled" value="true" />
+    <add key="DefaultOrganizationId" value="1" />
+    <add key="KudosAvailableToSendPerMonth" value="30" />
```

Comment:
This is kind of a breaking change for currently deployed Simoona instances, since it will override an old default of 20 with next deployment if no action is taken. Either remove this setting from default web.config (and provide it in web.release.config used for your deployments), or change it to an old default value - 20.

Also in general - Simoona is multi-tenant solution (at least main parts of it). For example main Visma instance is hosting it for two organizations/tenants currently (in the past there were even more). Providing values in web.config would set settings for all tenants, which is not good.

Maybe monthly kudos amount is not so critical thing, but in general this pattern should be avoided. For now, some of such settings are stored in `Organizations` database table.

Reply:
I agree that this is just a relocation of the previously hard coded constant into configuration.
Ideally this should be configurable from UI for each tenant. And could even be configurable per-role. 
But as a temporary measure to enable single-tenant users to configure it globally it should be OK.

Speaking about the setting in question: 
I assume that usually when updating to a new version users don't blindly use a new `Web.config`, since it can reset all of their settings back to default.

Unless they update the source and rebuild in which case only using smart merge would add this new line to the config.

Maybe I should change this to be a commented out line to communicate better that this setting is optional.

Reply:
> But as a temporary measure to enable single-tenant users to configure it globally it should be OK.

Such "temporary measures" become "permanent" right on time they are merged in. :)

> I assume that usually when updating to a new version users don't blindly use a new Web.config, since it can reset all of their settings back to default.

Everything what is pushed to remote repository accidentally or non-accidentally can be used blindly. 
I suggest just change the new value in web.config to a previous default value (which was hardcoded).
This won't introduce any unexpected behaviour for the ones who are not concerned about this change.

Reply:
OK, changed.